.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
   global:
     - PACKAGE="ocaml-migrate-parsetree"
     - PRE_INSTALL_HOOK="cd /home/opam/opam-repository && git pull origin master && opam update -u -y"
+    - TESTS=false
```

Comment:
To be honest, we can just entirely get rid of travis. But if you prefer, I can do that in a different PR.